outputs.tf

```diff
@@ -9,7 +9,7 @@ output "username" {
 }
 
 output "password" {
-  description = "The Fullstory password that can be used in the Fullstory app when configuring the Snowflake integration."
+  description = "The Fullstory password that can be used in the Fullstory app when configuring the Snowflake integration. Will be empty if `disable_password` is true."
```

Comment:
```suggestion
  description = "The password for the configured user that can be used in the Fullstory app when configuring the Snowflake integration. Will be empty if `disable_password` is true."
```

Reply:
committed, though had to rerun docs, so I did it on my machine.